README.md

```diff
@@ -327,7 +327,32 @@ For more detail, please refer to the shell script `./tools/config_tfserving.sh`.
 - Distributed inference is only supported when using Redis as Key-Value storage. 
 - Reference documents: https://www.tensorflow.org/tfx/serving/custom_op
 
-### With Triton(W.I.P)
+### With Triton
+When building the custom operations shared library it is important to
+use the same version of TensorFlow as is being used in Triton. You can
+find the TensorFlow version in the [Triton Release
+Notes](https://docs.nvidia.com/deeplearning/triton-inference-server/release-notes/index.html). A
+simple way to ensure you are using the correct version of TensorFlow
+is to use the [NGC TensorFlow
+container](https://ngc.nvidia.com/catalog/containers/nvidia:tensorflow)
+corresponding to the Triton container. For example, if you are using
+the 23.05 version of Triton, use the 23.05 version of the TensorFlow
```

Comment:
Can we have a matrix indicating the mapping relationship between the NGC TF and TF/TFRA versions?

Reply:
the versions relationship can be seen in [Triton Release
Notes](https://docs.nvidia.com/deeplearning/triton-inference-server/release-notes/index.html)